treeherder/log_parser/parsers.py

```diff
@@ -388,6 +388,7 @@ class ErrorParser(ParserBase):
         r"|^The web-page 'stop build' button was pressed"
         r"|.*\.js: line \d+, col \d+, Error -"
         r"|^\[taskcluster\] Error:"
+        r"|^\[[\w-]+:(?:error|exception)+\]"
```

Comment:
Doh! Thanks for catching that.

Is there an easy way to run these tests locally so I can check it out before committing more code?